packages/async-queue/package.json

```diff
@@ -19,8 +19,7 @@
 	"scripts": {
 		"test": "jest",
 		"lint": "eslint src tests --ext ts --fix -c ../../.eslintrc",
-		"build": "rollup -c",
-		"start": "yarn build -w",
+		"build": "tsup && tsc -b src",
```

Comment:
Doesn't this override...the tsup index.js?

Reply:
Wait, emitDeclarationOnly. Woops

Reply:
No because it's set to `emitDeclarationOnly`